packages/client/src/components/measurements/MeasurementStatisticsBelowChart.re

```diff
@@ -0,0 +1,131 @@
+[@react.component]  
+let make =  
+    (
+        ~cdf: ForetoldComponents.Types.Dist.t, 
+        ~measurable: Types.measurable
+    ) => {
+    
+    module Calculate = {
+        let toPdf = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ys =  cdf.ys;
+            let ysPdf = Array.mapi((index, element) => {
+            switch(index){
+                | 0 => element;
+                | someOtherIdex => element -. Array.get(ys,index-1);
+            }
+
+            }, ys); 
+            
+            ysPdf;
+        } 
+        let expectedValue = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ys= toPdf(cdf);
+            let result = Js.Array.reducei((accumulator, x, index) => (accumulator +. (x *. Array.get(ys,index)) ), 0.0, cdf.xs)
+            result;
+        } 
+
+        let median = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ys = cdf.ys;
+            let pos = Js.Array.findIndex( y => y>=0.5, ys);
+            pos==(-1) ? None : Some(cdf.xs[pos]);
+        }
+
+        let confidenceInterval = (cdf: ForetoldComponents.Types.Dist.t,p) => {
+            let ys = cdf.ys;
+            let low_p = ((100.0-.p)/.100.0)/.2.0;
+            let high_p = 1.0-.low_p;
+            
+            let low_value_index = Js.Array.findIndex( y => y>=low_p, ys);
+            let high_value_index = Js.Array.findIndex( y => y>=high_p, ys);
+
+            let low_value = Array.get(cdf.xs,low_value_index);
+            let high_value = Array.get(cdf.xs,high_value_index);
+
+            (low_value, high_value);
+        }      
+
+        let highestLikelihood = (cdf: ForetoldComponents.Types.Dist.t) => {
+            let ysPdf = toPdf(cdf) 
+            let indexOfMaxValue = Js.Array.reducei((iMax, x, i) => (x > Array.get(ysPdf,iMax))? i : iMax, 0, ysPdf);
```

Comment:
I think we can expect the ys and xs to be in order, so if you want the highest, just get the last.